### ISSUE TITLE
Question: Can I build my Own Ember Components

### ISSUE THREAD

**Issue by jrock2004** [empress/empress-blog#77]
So I am excited that this blog is built using ember. I am curious, could I create something like a twitter component and call that component in one of my blog posts? Or would I need to create my own template that I could call those? Just curious if I can write and use some ember code?

**Comment by elwayman02** [comment]
I don't think you can call it from within the markdown file (though it is possible to build that capability), but you could copy the `post.hbs` file from your blog template (e.g. Casper) and customize it to render additional components however you want!
